SECURITY.md

```diff
@@ -0,0 +1,44 @@
+# Security Policy
+
+The Solidity team and community take all security bugs in Solidity seriously.
+We appreciate your efforts and responsible disclosure and will make every effort to acknowledge your contributions.
+
+## Scope
+
+Bugs in the solidity repository are in scope.
+Bugs in third-party dependencies e.g., jsoncpp, boost etc. are not in scope unless they result in a solidity specific bug.
+
+Only bugs that have a demonstrable security impact on smart contracts are in scope.
+For example, a solidity program whose optimization is incorrect (e.g., leads to an incorrect output) qualifies as a security bug.
+Please note that the [rules][2] of the [Ethereum bounty program][1] have precedence over this security policy.
+
+## Supported Versions
+
+As a general rule, only the latest release gets security updates.
+Exceptions may be made when the current breaking release is relatively new, e.g. less than three months old.
+If you are reporting a bug, please state clearly the Solidity version(s) it affects.
+
+Example 1: Assuming current release is `0.6.3` and a security bug has been found in it that affects both `0.5.x` and `0.6.x` trees, we may not only patch `0.6.3` (the bug-fix release numbered `0.6.4`) but `0.5.x` as well (the bug-fix release numbered `0.5.(x+1)`).
+
+Example 2: Assuming current release is `0.6.25` and a security bug has been found in it, we may only patch `0.6.25` (in the bug-fix release numbered `0.6.26`) even if the bug affects a previous tree such as `0.5.x`.
+
```

Comment:
Should we also mention the bug list?

Reply:
Yes, that's a good idea. I'll draft a couple of sentences to that effect.

Reply:
Updated PR.